oag/ja-jp/content/topics/access-gateway/ref-arch/kerb-simp.htm

```diff
@@ -319,7 +319,7 @@ <h2>利点と欠点</h2>
                                                                         <li>フォールトトレランスを持たない</li>
                                                                         <li>負荷分散や高可用性の機能がない</li>
                                                                         <li>本番環境を意図していない</li>
-                                                                        <li>管理サーバーへのアクセスには、SSHポート22とHTTPSポート443を開く必要があります。詳細については、<a href="../about-oag-prereqs.htm" class="MCXref xref">「Access Gatewayデプロイメントの前提条件」</a>を参照してください。</li>
+                                                                        <li>管理サーバーへのアクセスには、SSHポート22とHTTPSポート443を開く必要があります。詳細については、「<a href="../about-oag-prereqs.htm" class="MCXref xref">Access Gatewayデプロイメントの前提条件</a>」を参照してください。</li>
```

Comment:
changes from [''link,,](x) to ''[link](x),, - just confirming if this is intentional?

Reply:
I'm guessing the translators decided that anchor tags should be surrounded by these JA-style quotation marks.